providers/efa/efa_io_defs.h

```diff
@@ -6,6 +6,12 @@
 #ifndef _EFA_IO_H_
 #define _EFA_IO_H_
 
+#define EFA_GET(ptr, type) \
+	((*(ptr) & type##_MASK) >> type##_SHIFT)
+
+#define EFA_SET(ptr, type, value) \
+	*(ptr) |= ((value) << type##_SHIFT) & type##_MASK
```

Comment:
Macros should be expressions not statements, see the checkpatch warning this triggers. Add a () around it.

Reply:
OK for the checkpatch ().
What do you suggest for the first part though?

Reply:
What first part? Put ({}) around the whole thing

Reply:
Done